.github/workflows/operator-integration-test.yml

```diff
@@ -68,17 +68,42 @@ jobs:
             echo "Pod is created test case: PASS"
           fi
 
-      - name: Test for default instrumentation resources
+      - name: Test for default instrumentation resources for Java
         run: |
           cd integration-tests/manifests
-          kubectl apply -f ./sample-deployment.yaml
+          kubectl apply -f ./sample-deployment-java.yaml
           sleep 60
           kubectl get pods -A 
           kubectl describe pods -n default
           cd ./cmd
-          go run validate_instrumentation_vars.go default default_instrumentation_env_variables.json
+          go run validate_instrumentation_vars.go default default_instrumentation_java_env_variables.json
 
-      - name: Test for defined instrumentation resources
+      - name: Test for defined instrumentation resources for Java
+        run: |
+          cd integration-tests/manifests
+          kubectl apply -f ./sample-instrumentation.yaml
+          sleep 60
+          kubectl rollout restart deployment nginx
+          sleep 60
+          kubectl get pods -A
+          kubectl describe pods -n default
+          cd ./cmd
+          go run validate_instrumentation_vars.go default ns_instrumentation_env_variables.json     
+
+      - name: Test for default instrumentation resources for python
+        run: |
+          cd integration-tests/manifests
```

Comment:
nit: I prefer to use absolute paths. So can just do `kubectl apply -f integration-tests/manifests/sample-deployment-python.yaml`

I know this was carry-over though